test/strategy.test.ts

```diff
@@ -78,74 +78,17 @@ describe.each(examples)(
       console.log(`Resolved in ${(getScoresTime / 1e3).toFixed(2)} sec.`);
     }, 2e4);
 
-    it('Should return an array of object with addresses', () => {
-      expect(scores).toBeTruthy();
-      // Check array
-      expect(Array.isArray(scores)).toBe(true);
-      // Check array contains a object
-      expect(typeof scores[0]).toBe('object');
-      // Check object contains at least one address from example.json
-      expect(Object.keys(scores[0]).length).toBeGreaterThanOrEqual(1);
-      expect(
-        Object.keys(scores[0]).some((address) =>
-          example.addresses
-            .map((v) => v.toLowerCase())
-            .includes(address.toLowerCase())
-        )
-      ).toBe(true);
-      // Check if all scores are numbers
-      expect(
-        Object.values(scores[0]).every((val) => typeof val === 'number')
-      ).toBe(true);
-    });
-
```

Comment:
HI @monish-nagre We should remove existing testcases, could you revert this file?

Reply:
Thanks for reply , yes i do